app/v2/notifications/post_notifications.py

```diff
@@ -201,14 +201,13 @@ def process_sms_or_email_notification(
         template_with_content=template_with_content
     )
 
-    if service.id in current_app.config.get('HIGH_VOLUME_SERVICE') and api_user.key_type == KEY_TYPE_NORMAL \
-       and notification_type == EMAIL_TYPE:
+    if service.id in current_app.config.get('HIGH_VOLUME_SERVICE') and api_user.key_type == KEY_TYPE_NORMAL:
         # Put GOV.UK Email notifications onto a queue
```

Comment:
Comment could be updated as this is no longer just GOV.UK Email and the final sentence about `we know that time team does not use GET requests` could be made more generic to just describe the behaviour change, for example

High volume services may not be able to immediately GET their notification, so should either be aware of this or not need to GET request their notifications, for example by relying on callbacks instead to get status updates.